lib/roll_slot.dart

```diff
@@ -64,28 +76,48 @@ class _RollSlotState extends State<RollSlot> {
 
   @override
   void dispose() {
-    _controller.dispose();
+    _infiniteScrollController.dispose();
     super.dispose();
   }
 
   @override
   Widget build(BuildContext context) {
     return AbsorbPointer(
-      child: ListWheelScrollView.useDelegate(
+      absorbing: false,
+      child: InfiniteCarousel.builder(
         physics: BouncingScrollPhysics(),
         itemExtent: widget.itemExtend,
-        diameterRatio: widget.diameterRation,
-        controller: _controller,
-        squeeze: widget.squeeze,
-        perspective: widget.perspective,
-        childDelegate: ListWheelChildLoopingListDelegate(
-          children: currentList.map((_widget) {
-            return Padding(
-              padding: widget.itemPadding,
-              child: _widget,
-            );
-          }).toList(),
-        ),
+        controller: _infiniteScrollController,
+        itemCount: widget.children.length,
+        axisDirection: Axis.vertical,
+        itemBuilder: (context, index, realIndex) {
+          if (widget.rollSlotController!.state.isStopped) {
+            if (realIndex == _stopIndex) {
+              centerTemporaryIndex = widget.rollSlotController!.centerIndex;
+              return widget.children[widget.rollSlotController!.centerIndex];
```

Comment:
consider using `centerTemporaryIndex` you created instead of `widget.rollSlotController!.centerIndex`